apps/model/model.js

```diff
@@ -743,9 +754,18 @@ async function deleteModel(name) {
       alert(err);
     } finally {
       if (status) {
-        alert('Deleted', name);
-        showInfo();
-        modelName.splice(modelName.indexOf(name.split('/').pop().split('_').splice(2).join('_').slice(0, -3)), 1);
```

Comment:
Hey @mgautam98 . I had added this line to remove the model name from ``` modelName ``` array so that that name could be reused once that model has been deleted. Is it not required ?

Reply:
This line is not needed anymore. After deletion I am calling initUIComponent which will fill the modelName array again.

Reply:
Oh sorry for the confusion. I missed that . Thanks for clearing that for me!